std_msgs/Header header

# free text definition of the meaning of the parameters below
string actionspace_type

# free parameters depending on parameterization type
float64[] action_parameters
